ansible/dev/group_vars/all.yml

```diff
@@ -14,3 +15,14 @@ ansible_user: "{{ lookup('env', 'USER') }}"
 gcp_default_labels:
   stackrox-ci: "false"
 
+
+# IBM Cloud setup
+# vpc_name: lei-vpc-toronto
+vpc_id: r038-c4f8e233-4b56-44bd-af71-140ebd1664e9
+# subnet_name: stackrox-ci-sn3
+subnet_id: 02s7-0fca7f4f-7e53-40d5-a087-f8cb5d7ab682
+zone: ca-tor-3
+# ssh_key_name: acs-sshkey
+ssh_key_id: r038-fb0260c7-c01d-45c8-8026-7d50042943b9
+# vsi_resource_group: stackrox-ci-resource-group
+vsi_resource_group_id: 1a33a6a9bd6e498f8115e9b1064bfa97 
```

Comment:
Will there be separate values for these vars for CI? If they're going to be static between dev and ci profiles, they should be moved to `group_vars/all.yml` otherwise we'll need these in `ci/group_vars/all.yml` as well

Reply:
They should be the same for both `ci` and `dev`. So moved them under `group_vars/all.yml`. Thanks for the suggestion.